packages/admin/cms-admin/src/dam/FileForm/EditFile.tsx

```diff
@@ -78,16 +77,12 @@ const useInitialValues = (id: string) => {
 
 const EditFile = ({ id }: EditFormProps): React.ReactElement => {
     const { match: scopeMatch } = useContentScope();
-    const persistedDamLocationApi = usePersistedDamLocation();
     const initialValues = useInitialValues(id);
     const file = initialValues.data?.damFile;
 
     if (initialValues.loading) {
         return <CircularProgress />;
     } else if (initialValues.error || file === undefined) {
-        // otherwise, the user always gets redirected to the broken file and is stuck there
-        persistedDamLocationApi?.reset();
```

Comment:
Is it safe to remove this reset? I haven't thought about that, sorry

Reply:
As long as we only use the persistence in the `ChooseFileDialog`, this is save. 

It was only needed in the DAM because this case only occurred in the file form (when persisting the DAM location while editing a file). That can't happen in the `ChooseFileDialog`.

I moved the `RedirectToPersistedDamLocation` from the `DamTable` to the `ChooseFileDialog` to signalize that it should only be used in the dialog.